ext/intl/tests/gregoriancalendar_set_date_error.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+IntlGregorianCalendar::setDate(): error cases
+--EXTENSIONS--
+intl
+--FILE--
+<?php
+ini_set("intl.error_level", E_WARNING);
```

Comment:
This ini setting seems to be useless? Moreover could use the ``--INI--`` test section